README.md

```diff
@@ -53,7 +53,9 @@ This will run the journal with its default settings. The default settings can be
 - Detailed tests under failure conditions are still missing.
 - Range deletion performance (i.e. `deleteMessages` up to a specified sequence number) depends on the extend of previous deletions
     - linearly increases with the number of tombstones generated by previous permanent deletions and drops to a minimum after compaction
-- Events by tag uses Cassandra Materialized Views which are a new feature that has yet to stabalise. Use at your own risk, see [here](https://lists.apache.org/thread.html/d81a61da48e1b872d7599df4edfa8e244d34cbd591a18539f724796f@%3Cdev.cassandra.apache.org%3E) for a recent discussion on the Cassandra dev mailing list.
+- For versions prior to 0.80 events by tag uses Cassandra Materialized Views which are a new feature that has yet to stabalise.
```

Comment:
was misspelled before already, but "stabilize"

Reply:
I beg to differ...

Reply:
I looked "stabalise" up and now I'm sure it is not what we want, also I will have to wash my eyes with soap.

Reply:
(the s I'm fine with)

Reply:
ohhhhh i am blind, sorry!